src/main/java/build/buildfarm/common/config/Cas.java

```diff
@@ -29,6 +29,9 @@ public enum TYPE {
   // GRPC
   private String target;
 
+  // Metrics
+  private boolean publishTtlMetric = false;
```

Comment:
shall we add this option to the example?

Reply:
added